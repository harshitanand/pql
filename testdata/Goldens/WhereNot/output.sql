SELECT * FROM "Tokens" WHERE NOT ("Kind" > 0);
