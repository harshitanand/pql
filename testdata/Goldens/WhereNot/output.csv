Kind,TokenConstant
-1,TokenError
